odoo/tools/misc.py

```diff
@@ -1708,3 +1708,26 @@ def street_split(street):
         'street_number': results[1].strip(),
         'street_number2': results[2],
     }
+
+
+def is_list_of(value, item_type):
+    """Return True if the given value is a list / tuple of the given type."""
+    if not isinstance(value, (list, tuple)):
+        return False
+
+    return all(isinstance(item, item_type) for item in value)
+
+
+def is_list(value, args):
```

Comment:
Not a fan of "is_list", would only keep "is_list_of" and either pass it [type * len(array)] either let it accept a single type or an array as first argument.

"is_list" makes it sound as if it's only going to check if it's a list.

Reply:
ok I removed `is_list`
But it would add more code in fields.py, so instead I added a keyword `exact=True` to separate both behavior, tell me what you think